app/assets/stylesheets/styles/admin.scss

```diff
@@ -0,0 +1,33 @@
+
+.container-large {
+  padding-left: 20em;
```

Comment:
For consistency, you should stick to one variable naming convention. It's preferable to use hyphens "-". Ensure you use this across all classes and ids.